cfgov/unprocessed/css/skip-nav.less

```diff
@@ -20,8 +20,13 @@
 }
 
 #skip-nav:focus {
-    .btn;
-    .btn__super;
+    .a-btn;
```

Comment:
Let's not forget about moving these into the markup to avoid the duplicate styles here.